internal/httpserve/route/upload.go

```diff
@@ -0,0 +1,37 @@
+package route
+
+import (
+	"net/http"
+
+	echo "github.com/theopenlane/echox"
+)
+
+var (
+	// uploadKeys are the keys that can be used to upload files in a multipart form
+	uploadKeys = []string{"uploadFile"}
```

Comment:
@golanglemonade shouldn't this use the config parameter vs. a static var?

Reply:
ah yep, missed this when I switched it to the config